poller-lambdas/src/pollers/ap/parseNitfContent.ts

```diff
@@ -0,0 +1,47 @@
+import type { HTMLElement } from 'node-html-parser';
+import { parse } from 'node-html-parser';
+
+export type ContentFromNitf = {
+	bodyContentHtml?: string;
+	issueDate?: string;
+	byline?: string;
+	headline?: string;
+	abstract?: string;
+	edMessage?: string;
+};
+
+export function nitfBlockToHtml(block: HTMLElement): string {
+	const clonedBlock = block.clone() as HTMLElement;
+	Array.from(clonedBlock.querySelectorAll('hl2')).forEach(
+		(hl2) => (hl2.tagName = 'h2'),
+	);
+	Array.from(clonedBlock.querySelectorAll('media,media-reference')).forEach(
+		(mediaTag) => mediaTag.remove(),
+	);
```

Comment:
It looks like node-html-parser already returns an array from `querySelectorAll`, so no need to construct another for the loop

```suggestion
	clonedBlock.querySelectorAll('hl2').forEach(
		(hl2) => (hl2.tagName = 'h2'),
	);
	clonedBlock.querySelectorAll('media,media-reference').forEach(
		(mediaTag) => mediaTag.remove(),
	);
```

but personally I find plain for-loops easier to follow for this kind of mutation

```suggestion
	for (const hl2 of clonedBlock.querySelectorAll('hl2')) {
		hl2.tagName = 'h2';
	}
	for (const mediaTag of clonedBlock.querySelectorAll('media,media-reference')) {
		mediaTag.remove();
	}
```

Reply:
oh nice! I'd assumed it returned a `NodeList` like the DOM method does 👍